cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/ShortestPathPlanningTest.scala

```diff
@@ -109,6 +109,10 @@ class ShortestPathPlanningTest extends DocumentingTest {
             |stream.
             |On top of this, the planner will issue an `AntiConditionalApply`, which will run the exhaustive search
             |if the path variable is pointing to `null` instead of a path.""")
+        p(
+          """An `ErrorPlan` operator will appear in the execution plan in cases where (i)
+            |`cypher.forbid_exhaustive_shortestpath` is set to `true`, and (ii) the fast algorithm is not able to fulfill the query.""".stripMargin
```

Comment:
Looks good, although _not able to fulfill the query_ might be misleading. The real thing is that shortestPath will give no results, and we cannot tell if that really means there are no shortestPaths, or whether it just failed to find any conforming to the predicates. Perhaps change the text to 'the fast algorithm was not able to find a shortestPath'. The difference is subtle, I think.

Reply:
I have pushed this change 👍